app/reducers/treatment.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
```

Comment:
I may be misunderstanding. You want to create a 'parent' onEnter that maintains the currentPlan (and its treatments) on our store for its children, e.g., plan, treatment, etc? That's ok by me. Didn't want to make that change to the architecture, hence my modularized approach for treatment.

Let me know what the conflicts are for counter. Happy to deconflict!